week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -7,8 +7,14 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {}
-function convertToBRL(price) {}
+function convertToUSD(price) {
+  let currency = (price * 1.39)+ 1/100;
+  return Math.round(currency);
+}
+function convertToBRL(price) {
+  let currency = (price * 4.86) + 1/100;
```

Comment:
Can you think of a way to unify the logic for adding 1% fee? So that next time the fee changes (say, to 2%), we only need to modify it in one place?